tests/__init__.py

```diff
@@ -46,9 +47,19 @@ def requeue_errors(self) -> None:
             doc_pair.error_next_try = 0
 
 
+def _basename(path):
+    """ Patch shutil._basename for pathlib compatibility """
```

Comment:
Could you add "TODO: remove when https://bugs.python.org/issue32689 is fixed (Python 3.7.3 or newer)"?